### DISCUSSION TITLE
How to pull out string from field with brackets

### DISCUSSION THREAD

**Discussion by sr320** [RobertsLab/resources#1087] [Q&A] [answered]
I have a file, two fields, space delimited..

```
lcl|NW_011934501.1_cds_XP_011426503.1_1 [gene=LOC105317034]
lcl|NW_011934501.1_cds_XP_011434216.1_2 [gene=LOC105317034]
lcl|NW_011934501.1_cds_XP_011445785.1_3 [gene=LOC105338483]
lcl|NW_011934505.1_cds_XP_019917900.1_4 [gene=LOC105346508]
lcl|NW_011934507.1_cds_XP_011423156.1_5 [gene=LOC105322465]
lcl|NW_011934512.1_cds_XP_011426108.1_6 [gene=LOC105327114]
```

and want it to clean up LOCID so it looks like 

```
lcl|NW_011934501.1_cds_XP_011426503.1_1 LOC105317034
lcl|NW_011934501.1_cds_XP_011434216.1_2 LOC105317034
```

I tried `sed 's/[gene=//g'` with both `"` and `'` around what I want to replace, but the bracket seems to be an issue.

**Comment by kubu4** [comment, marked as answer]
Using `sed` try escaping (`\`) the bracket.

**Reply by sr320** [reply]
That worked!

specifically `sed 's/\[gene=//g' | sed 's/\]//g'`

[more on subject](https://unix.stackexchange.com/questions/32907/what-characters-do-i-need-to-escape-when-using-sed-in-a-sh-script)
